Solutions/Zinc Open Source/Analytic Rules/ZincOctober2022_IP_Domain_Hash_IOC.yaml

```diff
@@ -1,8 +1,7 @@
 id: 95543d6d-f00d-4193-a63f-4edeefb7ec36
-name: Zinc Actor IOCs domains hashes IPs and useragent - October 2022
+name: "[Deprecated] - Zinc Actor IOCs domains hashes IPs and useragent - October 2022"
 description: | 
-  'Identifies a match across domainname, IP, hash and useragent IOCs related to an actor tracked by Microsoft as Zinc.
-  Reference: https://www.microsoft.com/security/blog/2022/09/29/zinc-weaponizing-open-source-software/'
+  'This query has been deprecated as the associated IoCs (Indicators of Compromise) are outdated and no longer relevant. To ensure effective threat detection, it is recommended to implement Microsoft's Threat Intelligence solution, which enables matching your log data with the most up-to-date IoCs generated by Microsoft. This solution can be installed from the Microsoft Sentinel Content Hub if not currently deployed. More details on the Content Hub can be found here: https://learn.microsoft.com/azure/sentinel/sentinel-solutions-deploy'
```

Comment:
Hi @praveenthepro, 
The description exceeds the 255 characters limit. Please shorten it to fit the limit. Thanks

Reply:
Updated the description within the 255 characters limit, Please review @v-atulyadav